modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/nabber.dm

```diff
@@ -0,0 +1,254 @@
+#define BASE_CLOTH_X_1 1
+#define BASE_CLOTH_Y_1 1
+
+#define NABBER_COLD_THRESHOLD_1 180
+#define NABBER_COLD_THRESHOLD_2 140
+#define NABBER_COLD_THRESHOLD_3 100
+
+#define NABBER_HEAT_THRESHOLD_1 300
+#define NABBER_HEAT_THRESHOLD_2 440
+#define NABBER_HEAT_THRESHOLD_3 600
+
+/datum/species/nabber
+	name = "Gigant armored sertenpid"
```

Comment:
this is an exceptionally long species name that i also think might be misspelled, please spell check your work